browser/ui/BUILD.gn

```diff
@@ -228,6 +228,14 @@ source_set("ui") {
       "toolbar/brave_bookmark_sub_menu_model.cc",
       "toolbar/brave_bookmark_sub_menu_model.h",
       "toolbar/brave_recent_tabs_sub_menu_model.h",
+      "webui/brave_education/brave_browser_command_handler.cc",
```

Comment:
I thought we discussed not having this be specific to brave education?

Reply:
At some point I moved it into the `brave_education` folder, but I can't remember why now. I'll move it back to `browser_command` as before.